### PR TITLE
Refactor/regex

### PR DESCRIPTION
**Pull request by leidyzulu**
edit_text_whit_mask
Adding the option mask on BaseEditTextFormField

### CODE REVIEW

File: test/src/main/res/layout/activity_baseedittext_mask_with_dashes.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            app:regex="\\d{3}-\\d{2}-\\d{4}"
+            app:mask="###-##-####"
+            app:input_type="number"
+            app:digits="0123456789-"
+            app:max_characters="11"
+            />
+
+
```
Comment (alejandrorios-condor) [inline test/src/main/res/layout/activity_baseedittext_mask_with_dashes.xml]: remove extra space

File: test/src/main/res/layout/activity_baseedittext_mask_with_dashes_and_points.xml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            app:regex="\\d{5}-\\d{4}\\."
+            app:mask="#####-####."
+            app:input_type="number"
+            app:digits="0123456789-."
+            app:max_characters="11"
+            />
+
+
```
Comment (alejandrorios-condor) [inline test/src/main/res/layout/activity_baseedittext_mask_with_dashes_and_points.xml]: remove extra space

File: test/src/main/java/co/condorlabs/customcomponents/test/MockActivity.kt [hunk 3]
```diff
@@ -18,6 +18,7 @@ package co.condorlabs.customcomponents.test
 
 import android.os.Bundle
 import androidx.appcompat.app.AppCompatActivity
+import kotlinx.android.synthetic.main.activity_mock.*
```
Comment (alejandrorios-condor) [inline test/src/main/java/co/condorlabs/customcomponents/test/MockActivity.kt]: remove wildcard

File: test/src/main/res/layout/activity_baseedittext_mask_with_dashes_show_error.xml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            app:regex="\\d{3}-\\d{2}-\\d{4}"
+            app:mask="###-##-##"
+            app:input_type="number"
+            app:digits="0123456789-"
+            app:hint="My text"
+            app:max_characters="9"
+            />
+
```
Comment (alejandrorios-condor) [inline test/src/main/res/layout/activity_baseedittext_mask_with_dashes_show_error.xml]: remove extra space

File: test/src/main/res/layout/activity_baseedittext_mask_with_dashes.xml (hunk 1 above)
Comment (leidyzulu) [reply test/src/main/res/layout/activity_baseedittext_mask_with_dashes.xml]: done

File: test/src/main/res/layout/activity_baseedittext_mask_with_dashes_and_points.xml (hunk 2 above)
Comment (leidyzulu) [reply test/src/main/res/layout/activity_baseedittext_mask_with_dashes_and_points.xml]: done

File: test/src/main/java/co/condorlabs/customcomponents/test/MockActivity.kt (hunk 3 above)
Comment (leidyzulu) [reply test/src/main/java/co/condorlabs/customcomponents/test/MockActivity.kt]: done

File: test/src/main/res/layout/activity_baseedittext_mask_with_dashes_show_error.xml (hunk 4 above)
Comment (leidyzulu) [reply test/src/main/res/layout/activity_baseedittext_mask_with_dashes_show_error.xml]: done

**Pull request merged**